debezium-core/src/main/java/io/debezium/data/Envelope.java

```diff
@@ -57,19 +61,20 @@ public enum Operation {
          */
         MESSAGE("m");
 
+        private static final Map<String, Operation> CODE_LOOKUP = Stream.of(Operation.values()).collect(
```

Comment:
I would add the reference as a comment here.

Reply:
Done.